src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -653,6 +662,21 @@ export default class CompleteResolution extends Mixins(CommonMixin, DateMixin, E
     this.updateResolutionStepValidationDetail()
   }
 
+  private async reRenderResolutionText (): Promise<void> {
+    this.$refs.resolutionTextRef.calculateInputHeight()
+  }
+
+  // Previously, the resolution text area would not render to the appropriate height relative to the amount of content
+  // when navigating from another step.  In hooking into the visibility change event on the resolution text area via the
+  // v-observe-visibility property, we are able to force a re-calculation of the text area height when a user navigates
+  // to the complete resolution step from another step for the first time. This results in the text area being rendered
+  // to the appropriate height.
```

Comment:
Thanks for the comment. This is exactly the kind of thing that needs a comment 👍 

So the component resizes when the user lands on it, or when the page renders?

Reply:
it resizes when the component is actually on the complete resolution step and the component is visible in the users view port